src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private static final Random RANDOM = new Random();
```

Comment:
According to the checklist, don't use static methods in your solution. Here, the RANDOM object is static, which is not recommended. Instead, you should create an instance of Random in the constructor of ColorSupplier or directly in the getRandomColor method if it's not used elsewhere.